File: evenOdd1.js
function EvenOdd(arr){
    let count = 0;
     let sum = 0;
     for (let i = 0;i < arr.length; i++){
       if (count % 2 == 0){
         sum += arr[i];
         count++;
       }else if (count % 2 != 0){
         sum *= arr[i];
         count++;
       }
   }
       return sum;
     }

File: even_odd1.py
def even_odd(arr):
    count,sum = 0,0
    for x in range(len(arr)):
        if count % 2 == 0:
            sum += arr[x]
            count += 1
        elif count % 2 != 0:
            sum *= arr[x]
            count += 1
    return sum